Yoast/wordpress-seo
# OpenGraph description term archives

## Summary


This PR can be summarized in the following changelog entry:

* Adds OpenGraph description meta tags for term archive pages.

## Relevant technical choices:

* Created a helper function to wrap the retrieving of a term description and used this in the twitter description too.
* Removed the empty check of the above helper for both the OG and twitter descriptions. As it is expected for the helper function to always return a string.
* Added a string helper class with the `wp_strip_all_tags` functionality.
* Fixes a bug where an empty meta tag could be output for the meta description and OG title. 

## Test instructions

This PR can be tested by following these steps:

You can test this with different taxonomies. I will outline it with the default categories below.
1. Set the default category meta description (`DEFAULT`): `Search appearance` > `Taxonomies` > `Categories`. Also, ensure the meta box is enabled.
1. Create a new category:
    1. Add a description: `DESCRIPTION`.
    1. Add a meta description: `META`.
    1. Add a facebook description  (under `Social`): `FACEBOOK`.
    1. Save the category.
1. Go to the frontend of this category.
1. Ensure the meta tag `og:description` has the content: `FACEBOOK`.
1. Remove the facebook description from the term.
1. Ensure the meta tag `og:description` has the content: `META`.
1. Remove the meta description from the term.
1. Ensure the meta tag `og:description` has the content: `DEFAULT`.
1. Remove the default meta description in the search appearance settings.
1. Ensure the meta tag `og:description` has the content: `DESCRIPTION`.
1. Remove the term's description.
1. Ensure the meta tag `og:description` is no longer there.

## UI changes

## Documentation

## Quality assurance


Fixes #
